### PR TITLE
add ping command

### PR DESCRIPTION
**Pull request by kddeisz**
To allow for writing something to ping lambda so we don't have to suffer the cold-start penalty

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -148,8 +148,12 @@ SlackBot.prototype.buildRouter = function () {
   return function (event, context, callback) {
     var foundCommand;
     var builtEvent = event;
-    builtEvent.body = qs.parse(builtEvent.body);
 
+    if (builtEvent.body === 'ping') {
+      return context.succeed('Ok');
+    }
```
Comment (mrdziuban) [inline index.js]: Will this exit fast enough? Calculations show that pinging every 5 minutes for a full day would cost $0.05/day ($17.55/year) for a lambda running with 1024MB of memory. Is that too much for the number of slackbots we're planning to use this with?

Comment (kddeisz) [inline index.js]: I'm not sure how we could make it faster, this is pretty much at the beginning of the handler. I think $17/year is worth not having to wait.

Comment (mrdziuban) [inline index.js]: whoops, did match wrong. it's actually $0.175/year, so it's negligible.

**Comment by mrdziuban** [comment]
Can we add documentation on this functionality?

**Comment by kddeisz** [comment]
@mrdziuban updated

**Comment by mrdziuban** [comment]
dope

Let's make this a minor version bump.

:shipit:

**Pull request merged**